test/Microsoft.ML.StaticPipelineTesting/Training.cs

```diff
@@ -749,5 +749,42 @@ public void FastTreeRanking()
             Assert.InRange(metrics.Ndcg[1], 36.5, 37);
             Assert.InRange(metrics.Ndcg[2], 36.5, 37);
         }
+
+        [Fact]
+        public void MultiClassNaiveBayesTrainer()
+        {
+            var env = new ConsoleEnvironment(seed: 0);
+            var dataPath = GetDataPath(TestDatasets.iris.trainFilename);
+            var dataSource = new MultiFileSource(dataPath);
+
+            var ctx = new MulticlassClassificationContext(env);
+            var reader = TextLoader.CreateReader(env,
+                c => (label: c.LoadText(0), features: c.LoadFloat(1, 4)));
+
+            MultiClassNaiveBayesPredictor pred = null;
```

Comment:
>MultiClassNaiveBayesPredictor [](start = 12, length = 29)

Should we expose Label and Feature histograms? Otherwise, I don't see any reason why we actually expose it.

Reply:
Created two public methods that will output a copy of the featurehistogram and labelhistogram

---
In reply to: [222122921](https://github.com/dotnet/machinelearning/pull/1111#discussion_r222122921) [](ancestors = 222122921)